packages/bpk-component-icon/gulpfile.js

```diff
@@ -29,4 +29,5 @@ require('@babel/register')({
   rootMode: 'upward',
 });
 
+// eslint-disable-next-line import/extensions
```

Comment:
Why is this needed? Can the import not be fixed?

Reply:
Will give it a go!